kotlin/usecases/subpub_app_android/Readme.md

```diff
@@ -6,7 +6,7 @@
 | ----------- | ----------- |
 | Description | Discusses how to develop a native Android application that has publish-subsciption functionality as well as the ability to translate messages by using the AWS SDK for Kotlin.     |
```

Comment:
publish-subsciption functionality as well as --> publish-subscription functionality and